### ISSUE TITLE
Can training continue from a previous trained model?

### ISSUE THREAD

**Issue by atalapan** [werner-duvaud/muzero-general#135]
Say I trained a game for 200,000 steps. A while later, I decide to train for 500,000 steps. Instead of doing 500,000 steps from scratch, can training start from the model previously trained on 200,000 steps? Thanks, this is fun and nicely written!

**Comment by JohnPPP** [comment]
Hi Atalapan,

Yes you can. Just load the previous model before starting w new training session.

Good luck,
João

**Comment by atalapan** [comment]
Great, thanks!

**Comment by dmtrung14** [comment]
How do I load a model? Is there a specific script for that? I'm using VMs, (colab / kaggle) by the way.

**Comment by dmtrung14** [comment]
> How do I load a model? Is there a specific script for that? I'm using VMs, (colab / kaggle) by the way.

Oh nvm then, I saw another issue closed with this. #181
